### ISSUE TITLE
Toolbox does not work in new Firefox installations

### ISSUE THREAD

**Issue by hez01** [B-i-t-K/PwnFox#16]
Hi. The Pwnfox's Toolbox (tried 1.0.4 and 1.0.3 versions) does not work on new installations of Firefox (Using Firefox 107.0 (64-bit) on Ubuntu).
It still works on my other profile (same installation), but does not on new installations or profiles. I am attaching an error from browser's console (not from F12, but from CTRL+SHIFT+J).
![Screenshot from 2022-11-20 13-21-43](https://user-images.githubusercontent.com/103249837/202899479-19d1f2e1-4f72-430c-9e07-bb8dd161b925.png)
Steps to reproduce:
1. Install new firefox (or create a new profile via about:profiles);
2. Install PwnFox addon;
3. Create a toolbox script (set in something like `console.log('asd')`);
4. Enable toolbox and PwnFox;
5. Navigate to any website;
6. Inspect Console from dev tools (F12 > console tab) - there are no logged entries.

**Comment by D3vil0p3r** [comment]
@hez01 is this issue still present?

@B-i-t-K PwnFox is still a maintained project?

Thanks

**Comment by hez01** [comment]
Yep, the issue is still there. Tried even on Windows - also does not work.
